scripts/issues.py

```
@@ -0,0 +1,366 @@
+@lru_cache
+def _bin(name: str) -> str:
+    path = shutil.which(name)
+    if path is None:
+        typer.echo(f"Error: {name} not found on PATH.", err=True)
+        raise typer.Exit(1)
+    return path
+
+
+@lru_cache
+def _repo_root() -> Path:
+    result = subprocess.run(
```

Ruff (0.15.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)